pine-a64-plus/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.img
+*.img.*
```

Comment:
move this file to root in a sepearte commit

Reply:
removed from commit